packages/data-service/package.json

```diff
@@ -39,13 +39,13 @@
     "lint": "npm run eslint . && npm run prettier -- --check .",
     "check": "npm run lint && npm run depcheck",
     "check-ci": "npm run check",
-    "pretest": "testserver start dataservice --port=27021",
+    "pretest": "node ../../scripts/testserver.js start dataservice --port=27021",
```

Comment:
You should be able to do `compass-scripts testserver` instead by the way (handled by `./scripts/cli.js`) which reads a bit easier